.pre-commit-config.yaml

```
@@ -47,3 +47,11 @@ repos:
         additional_dependencies: ['pep8-naming']
         # Ignore all format-related checks as Black takes care of those.
         args: ['--ignore', 'E2,W5', '--select', 'E,W,F,N', '--max-line-length=120']
+
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.971
+    hooks:
+      - id: mypy
+        exclude: '^(docs|tasks|tests)|setup\.py'
+        args: [--ignore-missing-imports]
+        additional_dependencies: [attrs~=19.3, types-all, pydantic]
```

```suggestion
        additional_dependencies: 
          - attrs~=19.3
          - types-all
          - pydantic
```        
I find array more readable this way, but that's a matter of taste...